main.rb

```diff
@@ -9,7 +16,7 @@ def handle_menu_choice(choice, app)
     5 => { description: 'Create a book', action: -> { app.create_book } },
     6 => { description: 'Create a rental', action: -> { app.create_rental } },
     7 => { description: 'List rentals for a person', action: -> { app.list_rentals_for_person } },
-    8 => { description: 'Quit', action: -> { exit_app } }
+    8 => { description: 'Quit', action: -> { exit_app(app) } }
```

Comment:
You have done a good job in setting up the console, however, when I quit the app, I get an error thrown on the console and I think it's with regards to the book title. Here is a screenshot to show the error I got. Consider addressing this in order to make the persistence of user data work smoothly when there is nothing yet saved.

<img width="708" alt="error" src="https://github.com/tomdan-ai/OPP-School-Library/assets/84930355/b5f3f3e1-261c-479d-8aa5-dffc3598e180">

Reply:
The error you have doesnt exist in mine, I believe it may be caused by an external issue